src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");

```

Comment:
Similarly, the stop message should be more informative. Instead of 'Excavator stopped working.', consider 'Excavator has stopped working.', to maintain consistency and clarity.